Framework/PythonInterface/mantid/_testing/AssertAlmostEqual.py

```diff
@@ -8,7 +8,7 @@
 from mantid.simpleapi import CompareWorkspaces
 
 
-def assert_almost_equal(Workspace1, Workspace2, rtol=0.0, atol=0.0):
+def assert_almost_equal(Workspace1, Workspace2, **kwargs):
     """
     Raises an assertion error if two workspaces are not within specified tolerance.
 
```

Comment:
You should document the `kwargs` by linking to [CompareWorkspaces](https://docs.mantidproject.org/nightly/algorithms/CompareWorkspaces-v1.html) so people know what options they have.